ws-client/src/client.rs

```diff
@@ -488,6 +478,17 @@ async fn send_unsubscribe_request(
 	request: RequestMessage,
 ) {
 	if let Err(e) = sender.start_request(request, manager).await {
-		log::error!("send unsubscribe request failed: {:?}", e);
+		log::error!("Send unsubscribe request failed: {:?}", e);
 	}
 }
+
+fn build_unsubscribe_request(
+	manager: &mut RequestManager,
+	req_id: u64,
+	sub_id: SubscriptionId,
+) -> Option<RequestMessage> {
+	let (_, unsub) = manager.remove_subscription(req_id, sub_id.clone())?;
```

Comment:
This `sub_id.clone()` is a bit unfortunate.

Reply:
Yeah, it's because the `RequestManager` is using the HashMap entry api internally.

Clone should be cheap though, `u64` or `String`